### PR TITLE
PageBanner: Move page banner above navbar

### PR DESCRIPTION
**Pull request by torkelo**
Moves page banner so that its above everything 

Screenshot from 2021-11-12 13-34-17

### CODE REVIEW

File: packages/grafana-ui/src/themes/GlobalStyles/page.ts [hunk 1]
```diff
@@ -7,13 +7,18 @@ export function getPageStyles(theme: GrafanaTheme2) {
 
   return css`
     .grafana-app {
+      position: relative;
       display: flex;
-      align-items: stretch;
-      position: absolute;
-      width: 100%;
+      flex-direction: column;
+      flex-grow: 1;
```
Comment (jackw) [inline packages/grafana-ui/src/themes/GlobalStyles/page.ts:13]: I think we can remove this as `flex: 1 1 0;` on line 15 will override it.

**Review by jackw — APPROVED** [review]
LGTM! 🚀

File: packages/grafana-ui/src/themes/GlobalStyles/page.ts (hunk 1 above)
Comment (mckn) [reply packages/grafana-ui/src/themes/GlobalStyles/page.ts:13]: +1

**Review by mckn — APPROVED** [review]
LGTM!

**Comment by tumaranja** [comment]
Hi all, x-posting from Slack: was looking at the design after a while with fresh eyes and thought there was room for improvement. Just asked the team for feedback, if you also have some thoughts on this happy to hear them: https://raintank-corp.slack.com/archives/C0F5GDR4P/p1636970996122900

**Comment by torkelo** [comment]
ok, closing this as we only need to remove the padding in the banner component (separate PR in enterprise)

**Comment by tumaranja** [comment]
@torkelo Sounds good, here is the [updated v2 banner in Figma](https://www.figma.com/file/7E4VEb2IRmdsjQIUlDTwxb/Cloud-Homepage?node-id=1912%3A17993)

**Pull request closed without merging**